### PR TITLE
152601157 error snackbar

### PR DESCRIPTION
**Pull request by rckopteri**
Move snackbar from top to bottom
Show red snackbar when errors
Show green when success

Snackbar is shown on deleting service, adding/modifying service and adding operator.

### CODE REVIEW

File: ote/src/cljs/ote/app/controller/transport_service.cljs [hunk 1]
```diff
@@ -11,6 +11,16 @@
             [ote.localization :refer [tr tr-key]]
             [ote.app.controller.place-search :as place-search]))
 
+(defn clean-app-state [app]
+      (let [transport-service (get app :transport-service)
```
Comment (tatut) [inline ote/src/cljs/ote/app/controller/transport_service.cljs]: Why is this function needed? It seems convoluted, can't we just reset service to `nil` or empty map `{}`.

File: ote/resources/public/language/sv.edn [hunk 2]
```diff
@@ -540,6 +540,9 @@
                :hours    "Tuntia"
                :days     "Päivää"
                :seconds  "Sekuntia"}
+  :save-failed                                "Tallennettaessa sattui virhe. Tarkista lomakkeen tiedot"
+  :delete-service-success                     "Palvelu poistettiin onnistuneesti."
+  :delete-service-error                       "Palvelun poisto epäonnistui. Koita hetken päästä uudestaan."
```
Comment (tatut) [inline ote/resources/public/language/sv.edn]: Colloquial term, use "Yritä" instead

File: ote/resources/public/language/sv.edn [hunk 3]
```diff
@@ -540,6 +540,9 @@
                :hours    "Tuntia"
                :days     "Päivää"
                :seconds  "Sekuntia"}
+  :save-failed                                "Tallennettaessa sattui virhe. Tarkista lomakkeen tiedot"
```
Comment (tatut) [inline ote/resources/public/language/sv.edn]: Phrasing blames the user, perhaps we need a link to open feedback form when an error is shown.

File: ote/src/cljs/ote/app/controller/transport_service.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/transport_service.cljs]: Maybe the function name is confusing. This cleans up transport-service object for the form. But it doesn't remove it completely. 
I will rename it.

**Comment by rckopteri** [comment]
Changes made

**Pull request merged**